lms/djangoapps/bookmarks/models.py

```diff
@@ -60,12 +123,63 @@ def get_path(block):
         Returns:
             list of dicts of the form {'usage_id': <usage_id>, 'display_name': <display_name>}.
```

Comment:
Please make these named tuples. Generally speaking, if you have a dict with a fixed set of keys, it's better to use a named tuple so that no one accidentally mutates the dictionary, and so that you could upgrade to a class in the future.